deployment/deployment.yml

```diff
@@ -181,11 +214,42 @@ spec:
               name: iam-staging
               key: DB_PASSWORD
         - name: FEAT_TOGGLE_FIREBASE
-          value:
+          value: "1"
+        - name: FIREBASE_PROJECT_ID
+          value: dd-decaf-cfbf6
+        - name: FIREBASE_CLIENT_CERT_URL
+          valueFrom:
+            secretKeyRef:
+              name: iam-staging
+              key: FIREBASE_CLIENT_CERT_URL
+        - name: FIREBASE_CLIENT_EMAIL
+          valueFrom:
+            secretKeyRef:
+              name: iam-staging
+              key: FIREBASE_CLIENT_EMAIL
+        - name: FIREBASE_CLIENT_ID
+          valueFrom:
+            secretKeyRef:
+              name: iam-staging
+              key: FIREBASE_CLIENT_ID
+        - name: FIREBASE_PRIVATE_KEY
+          valueFrom:
+            secretKeyRef:
+              name: iam-staging
+              key: FIREBASE_PRIVATE_KEY
+        - name: FIREBASE_PRIVATE_KEY_ID
+          valueFrom:
+            secretKeyRef:
+              name: iam-staging
+              key: FIREBASE_PRIVATE_KEY_ID
+        - name: prometheus_multiproc_dir
+          value: /prometheus-client
```

Comment:
It is to read the secrets on k8s??

Reply:
Yes the firebase values from the secrets, and `prometheus_multiproc_dir` is for gunicorn multiprocess support in the client which exposes the metrics ([docs here](https://github.com/prometheus/client_python#multiprocess-mode-gunicorn)).